{{cookiecutter.project_shortname}}/code/{{cookiecutter.project_package}}/views.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2021 CERN.
+# Copyright (C) 2019-2022 Northwestern University.
+# Copyright (C)      2021 TU Wien.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Additional views."""
+
+from flask import Blueprint
+
+#
+# Registration
+#
+def create_blueprint(app):
```

Comment:
this works without registering it in the app ?

Reply:
it's registered in the setup.cfg